src/types/websockets.ts

```diff
@@ -755,6 +755,9 @@ export interface WsMessageFuturesUserDataOrderTradeUpdateEventRaw
     pP: boolean; // ignore
     si: numberInString; // ignore
     ss: numberInString; // ignore
+    V: string; 
+    pm: string;
+    gtd: number;
```

Comment:
Is this goodTillDate? Can you add it to the beautifier if it isn't there already?

Reply:
Yep, missed that haha it isnt added now but it was missing.